### PR TITLE
Header #18 done

### PR DESCRIPTION
**Pull request by MadhushaPrasad**

### CODE REVIEW

**Comment by Shehanka** [comment]
@MadhushaPrasad can u fix the conflict

File: components/common/navbar/Navbar.tsx [hunk 1]
```diff
@@ -1,42 +1,168 @@
 import Link from 'next/link'
+import Image from 'next/image'
+import fcscLogo from '../../../public/logo/fcsc.png'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import { AnimatePresence, motion } from 'framer-motion'
+
+// animations
```
Comment (Shehanka) [inline components/common/navbar/Navbar.tsx:8]: remove these comments

File: components/common/navbar/Navbar.tsx [hunk 2]
```diff
@@ -1,42 +1,168 @@
...
+import {
+  mobileNavVariants,
+  navElementsVariants,
+} from '../../../animations/index'
+
+// icons
```
Comment (Shehanka) [inline components/common/navbar/Navbar.tsx:14]: this too

**Comment by MadhushaPrasad** [comment]
okay

File: components/common/navbar/Navbar.tsx [hunk 3]
```diff
@@ -1,42 +1,168 @@
...
+            >
+              <div
+                className="absolute top-4 right-4 cursor-pointer"
+                onClick={toggleNav}
+              >
+                {isOpen ? (
+                  <RiCloseFill className="h-6 w-6 hover:text-gray-light text-fcsc-blue transition ease-in" />
+                ) : (
+                  ''
+                )}
```
Comment (Shehanka) [inline components/common/navbar/Navbar.tsx:94]: use `&` like this `isOpen && <RiCloseFill className="h-6 w-6 hover:text-gray-light text-fcsc-blue transition ease-in" />`

File: pages/community.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+//components
+import Notice from '../components/carousel/notice/Notice'
+import NoticeCarousel from '../components/carousel/notice/NoticeCarousel'
+
+//images
+import background from '../public/events/BG.png'
+import noticeboard from '../public/notices/noticeboard.png'
+import notice from '../public/notices/notice.png'
+import Announcements from '../components/announcements/Announcements'
+import { QAProps } from '../components/common/QA/QAComponent'
```
Comment (Shehanka) [inline pages/community.tsx:13]: remove comments

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Review by sithumonline — APPROVED** [review]
LGTM

**Review by Shehanka — APPROVED** [review]

**Pull request merged**